packages/tools/devtools/devtools-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -33,6 +40,10 @@ export interface AudienceHistoryTableProps {
 export function AudienceHistoryTable(props: AudienceHistoryTableProps): React.ReactElement {
 	const { audienceHistoryItems } = props;
 
+	const TOOLTIP_FLUIDCLIENTGUID =
```

Comment:
Let's not use UPPER_SNAKE_CASE for inline constants. I would recommend just calling this `clientIdTooltipText` or something.

Reply:
Didn't realize the pattern has changed from the previous PR that I used as a reference.  https://github.com/microsoft/FluidFramework/pull/15157/files which uses UPPER_SNAKE_CASE for tooltips' contents. 

Thank you for letting me know 👍 .

Reply:
Oh, I'm sorry. I missed that we had been using that case there. Yeah, to my knowledge we don't encourage using that case anywhere, though I'm sure there are deviations.